src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -215,6 +215,28 @@ const testSuite = () => {
       const image = await screenshot()
       expect(image).toMatchImageSnapshot()
     })
+
+    it('drop hover after table', async () => {
```

Comment:
Let's move this test into the `drag` describe block.

Reply:
I think this test is safe to remove now, since it has been copied to the drag describe block below.